src/NLPModels.jl

```diff
@@ -427,12 +427,20 @@ end
 
 function jth_hprod! end
 
+"""
+   gHv = ghjvprod(nlp, x, g, v)
+Return the vector whose i-th component is g' ∇²cᵢ(x) v.
```

Comment:
Don’t we need a blank line before the `Return` so the markdown is formatted correctly?

Reply:
You are right, thanks !